### PR TITLE
734 update contributing

### PR DESCRIPTION
**Pull request by thomas-senechal**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,157 +1,189 @@
...
 
-  - Ubuntu:
-    - Run the following command to update the package lists:
 
-        ```bash
-        sudo apt update
+## Reviewing Pull Requests
 
-        ```
+Reviewing pull requests is an excellent way to contribute to the project. It helps us ensure that the codebase is well-maintained and that new features and bug fixes are properly tested and documented. If you have the time and skills to review pull requests, please consider doing so.
```
Comment (SlnPons) [inline CONTRIBUTING.md:44]: You could add the link to the github pull request menu?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,157 +1,189 @@
...
 
-    - Run the following command to install Node.js and NPM:
 
-        ```bash
-        sudo apt install nodejs npm
-        ```
+## Contributing fixes & features
```
Comment (SlnPons) [inline CONTRIBUTING.md:47]:
```suggestion
## Contributing to fixes & features
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,157 +1,189 @@
...
 
-- Install Go: Go is required to build and run Thyra. You can install Go by following the instructions at [https://go.dev/doc/install](https://go.dev/doc/install).
+To ensure a smooth collaboration and avoid duplication of efforts, please follow these guidelines:
 
-- Install Swagger: Thyra uses Swagger to generate code from the API documentation. You can install Swagger by running the following command:
+1. Before starting work on a new feature or bug fix, check the project's GitHub repository for existing issues.
```
Comment (SlnPons) [inline CONTRIBUTING.md:51]: Add a note to specify that they can start looking for "good first issue" flag and tackle any of this "easy issue". WDYT?

File: CONTRIBUTING.md (hunk 2 above)
Comment (SlnPons) [reply CONTRIBUTING.md:51]: It's just that in English, you contribute TO sth. You can't have contribute alone except if to don't refer to something but somebody.

**Review by SlnPons — APPROVED** [review]

**Review by B-Naoufal — APPROVED** [review]

**Pull request merged**